infra/terraform/modules/data_access/teams/teams.py

```diff
@@ -213,4 +214,19 @@ def detach_policy_from_user(policy_arn, user_name):
 
 
 def bucket_name(slug):
```

Comment:
Rather than repeating this function in each lambda function subdirectory, we should probably create a python package with all our common code that can be imported.